elmslie-android/src/main/java/vivid/money/elmslie/android/manager/ViewBasedElmManager.kt

```diff
@@ -0,0 +1,31 @@
+package vivid.money.elmslie.android.manager
+
+import android.app.Activity
+import androidx.lifecycle.Lifecycle
+import vivid.money.elmslie.android.storeholder.StoreHolder
+import vivid.money.elmslie.android.storestarter.ViewBasedStoreStarter
+import vivid.money.elmslie.core.store.Store
+
+class ViewBasedElmManager<Event : Any, Effect : Any, State : Any>(
```

Comment:
We can use the same naming as Compose does. For ViewBased things they use "Android" prefix)

Reply:
ViewBased will be used in ComposeFragments too, as for me "Android" prefix is a little bit confusing. Also, there will be one more ElmManager for compose functions only.
ComposeElmManager and ViewBasedElmManager - looks nice for me.